### ISSUE TITLE
Upgrades/Installs fail due to corrupt lxc image download

### ISSUE THREAD

**Issue by claco** [rcbops/rpc-openstack#315]
More information here: https://gist.github.com/nrb/602d11631820f4f670f5

Upgrades, and installations, occasionally fail with `Unexpected error when accessing exploded file: [Errno 2] No such file or directory` error:

```
TASK: [lxc_hosts | Move lxc cached image into place] **************************
failed: [578134-storage02] => (item={'url': u'http://rpc-repo.rackspace.com/container_images/rpc-trusty-container.tgz', 'name': 'trusty.tgz'}) => {"failed": true, "item": {"name$
: "trusty.tgz", "url": "http://rpc-repo.rackspace.com/container_images/rpc-trusty-container.tgz"}}
msg: Unexpected error when accessing exploded file: [Errno 2] No such file or directory: '/var/cache/lxc/trusty/rootfs-amd64/etc/ssl/certs/NetLock_Arany_=Class_Gold=_F\\305\\221$
an\\303\\272s\\303\\255tv\\303\\241ny.pem'
```

**Comment by odyssey4me** [comment]
Every single download from 10 hosts was corrupted? This is 10 downloads from an internet source in parallel, so I smell a rat!

**Comment by odyssey4me** [comment]
See https://bugs.launchpad.net/openstack-ansible/+bug/1482091 and https://review.openstack.org/209788

**Comment by mancdaz** [comment]
I would argue that this and the upstream patch are non essential fixes. Nice to haves.

**Comment by prometheanfire** [comment]
fixed upstream, have we repointed the submodule to fix this yet?

**Comment by stevelle** [comment]
yes.

commit 0ccf11eeddaad8b8f4b53e3a7cf3f33f81d208ee from osad is in the submodule's tree which is currently sync'd to 6f36c36b0cc8d186637cd7605371288edf5c0562

**Comment by stevelle** [comment]
nevermind

**Comment by claco** [comment]
Ha. So that fix doesn't prevent the issue (corrupted download), it just surfaces it.  With that in mind, and that we haven't seen this since the dupe ip fix went in, I'm going to lower the priority a bit.

**Comment by stevelle** [comment]
I had a bad sync of my subrepo, checked more carefully and doesn't appear to have made it into rpc_openstack yet. Either way, not going to argue that it can be lower priority.

**Comment by BjoernT** [comment]
There are still issues around LXC cache image handling but they are all OSA upstream https://bugs.launchpad.net/openstack-ansible/+bug/1539236 for example. I don't know if we need discuss this with this bug
